### DISCUSSION TITLE
Idee nuova versione

### DISCUSSION THREAD

**Discussion by luigirotondi2009** [iisgiua/giuaschool#705] [Idee e proposte]
Salve, nel nostro istituto servono i voti impostati cosí. Es. g = giustifica i = impreparato s = sufficiente b = buono.
Quale file dovremmo modificare, oppure  far uscire questa nuova  funzione nella prossima versione?

**Comment by trinko** [comment]
No, non è prevista questa funzione, anche perché ritengo la si possa già gestire inserendo del testo nel campo GIUDIZIO senza inserire un voto.

In ogni caso, per eventuali modifiche, vedi la discussione #641

**Reply by luigirotondi2009** [reply]
Va bene! Ma in ogni caso ne abbiamo la necessità, come possiamo svolgere questa operazione dell'aggiunta voti nella nostra maniera?

**Comment by trinko** [comment]
> Va bene! Ma in ogni caso ne abbiamo la necessità, come possiamo svolgere questa operazione dell'aggiunta voti nella nostra maniera?

Ho già risposto: vedi la discussione #641  dove è stato affrontato il problema.
